src/PrestaShopBundle/Service/DataUpdater/Admin/ProductInterface.php

```diff
@@ -63,7 +63,7 @@ public function deleteProductIdList(array $productIdList);
      *
      * @throws \PrestaShopBundle\Exception\UpdateProductException if duplication failed
      *
-     * @return bool true when succeed
+     * @return array|bool duplicates products list or false when all failed
```

Comment:
Finally see you change the interface :sweat_smile: 

But it's not sure it's the right way to do it because if we plan to refactor it, we will not be able to use multiple return types with the strict typing :(